src/OpenApi.Extensions/Transformers/Abstracts/XmlTransformer.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Martin Costello, 2024. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+using System.Reflection;
+using System.Xml;
+using System.Xml.XPath;
+
+namespace MartinCostello.OpenApi.Transformers.Abstracts;
```

Comment:
Rather than this be the only class in "Abstracts", can you just move this up a level to the parent please?